util/src/test/java/tk/zielony/randomdata/util/StringIPAddressGeneratorTest.java

```diff
@@ -11,21 +9,11 @@
     @Test
     public void generationIsCorrect() throws Exception {
         StringIPAddressGenerator ipAddressGenerator = new StringIPAddressGenerator();
-        for (StringIPAddressGenerator.classRange range : StringIPAddressGenerator.classRange.values()) {
+        for (StringIPAddressGenerator.ClassRange range : StringIPAddressGenerator.ClassRange.values()) {
             ipAddressGenerator.setClassRange(range);
-            String ipAddress = ipAddressGenerator.nextInternal(null);
+            String ipAddress = ipAddressGenerator.next();
             int initialSection = Integer.parseInt(ipAddress.substring(0, ipAddress.indexOf('.')));
-            switch(range) {
-                case A:
-                    assertEquals(true, 191 >= initialSection && initialSection >= 128);
-                    break;
-                case B:
-                    assertEquals(true, 223 >= initialSection && initialSection >= 192);
-                    break;
-                case C:
-                    assertEquals(true, 127 >= initialSection && initialSection >= 1);
-                    break;
-            }
+            assertEquals(true, range.max >= initialSection && initialSection >= range.min);
```

Comment:
min and max should be read-only. Please make them private, add accessors and use them